drivers/tty/serial/serial_core.c

```diff
@@ -3440,6 +3457,10 @@ int serial_core_register_port(struct uart_driver *drv, struct uart_port *port)
 err_unregister_ctrl_dev:
 	serial_base_ctrl_device_remove(new_ctrl_dev);
 
+	/* Delete the anon platform_device */
```

Comment:
Nitpick: this comment doesn't add any information which is not already obvious from the code. I would remove it (or expand it into something more useful if you so desire).

For more details, see the "8) Commenting" section of: https://www.kernel.org/doc/html/latest/process/coding-style.html

Reply:
Comment removed